requirements/macos-latest-3.10.txt

```diff
@@ -138,8 +138,6 @@ docker[ssh]==6.1.3
     # via
     #   docker-compose
     #   testcontainers-core
-docker-compose==1.29.2
```

Comment:
I don't think these files should be manually edited, since they contain the following comment:
```
# This file is autogenerated by pip-compile with Python 3.10
# by the following command:
#
#    pip-compile --output-file=requirements.txt --resolver=backtracking
```

Reply:
That is true but I cannot auto generate this with the required os/env.